client/ngapp/views/about.html

```diff
@@ -23,10 +23,10 @@
     <p>
     To contact the OONI team send an email to <a href="mailto:contact [AT] ooni.io">contact [AT] ooni.io</a>.
```

Comment:
While we are at it we should also change `contact@ooni.io` to `contact@openobservatory.org`